components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -117,14 +156,40 @@ - (void)setupTabBarColors {
 
 - (nonnull NSArray *)constructExampleViewControllers {
   NSBundle *bundle = [NSBundle bundleForClass:[TabBarViewControllerExample class]];
+  MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
+  buttonScheme.colorScheme = self.colorScheme;
+  buttonScheme.typographyScheme = self.typographyScheme;
+
   TBVCSampleViewController *child1 =
       [TBVCSampleViewController sampleWithTitle:@"One" color:UIColor.redColor];
+
   UIColor *blue = [UIColor colorWithRed:0x3A / 255.f green:0x56 / 255.f blue:0xFF / 255.f alpha:1];
   TBVCSampleViewController *child2 = [TBVCSampleViewController sampleWithTitle:@"Two" color:blue];
+  __weak TabBarViewControllerExample *weakSelf = self;
+  [child2 addMDCButtonWithFrame:CGRectMake(10, 120, 300, 40)
+                   buttonScheme:buttonScheme
+                          title:@"Push and Hide Tab"
+                    actionBlock:^{
+                      TabBarViewControllerExample *strongSelf = weakSelf;
+                      TBVCSampleViewController *vc =
+                      [TBVCSampleViewController sampleWithTitle:@"Push&Hide" color:UIColor.grayColor];
+                      vc.colorScheme = self.colorScheme;
+                      vc.typographyScheme = self.typographyScheme;
+                      [strongSelf.navigationController pushViewController:vc animated:YES];
+                    }];
+
   UIImage *starImage =
       [UIImage imageNamed:@"TabBarDemo_ic_star" inBundle:bundle compatibleWithTraitCollection:nil];
   TBVCSampleViewController *child3 =
       [TBVCSampleViewController sampleWithTitle:@"Three" color:UIColor.blueColor icon:starImage];
+  [child3 addMDCButtonWithFrame:CGRectMake(10, 120, 300, 40)
+                   buttonScheme:buttonScheme
+                          title:@"Toggle Tab Bar"
+                    actionBlock:^{
+                      TabBarViewControllerExample *strongSelf = weakSelf;
+                      [strongSelf setTabBarHidden:!strongSelf.tabBarHidden animated:YES];
```

Comment:
Since this is a property, please use the dot syntax.

```objc
strongSelf.tabBarHidden = !self.isTabBarHidden;
```

Reply:
I kept the original behavior since it wanted to animate out the tab bar. Would it be okay if I change the behavior?